bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller_test.go

```diff
@@ -1441,6 +1441,154 @@ func TestBootstrapTokenRotationMachinePool(t *testing.T) {
 	g.Expect(foundNew).To(BeTrue())
 }
 
+func TestBootstrapTokenRefreshIfTokenSecretCleanedMachine(t *testing.T) {
```

Comment:
Can we have a single test with two test run? (se we can have a better test description)
```suggestion
func TestBootstrapTokenRefresh(t *testing.T) {
   t.Run("should not recreate the token for Machines", func(t *testing.T) {
      ...
   })
   t.Run("should recreate the token for MachinePools", func(t *testing.T)  
      ...
   })

}
```

Reply:
done